src/backend/ai-reviewer-api/src/test/java/ca/bc/gov/open/jag/aireviewerapi/document/documentsApiDelegateImpl/DocumentEventTest.java

```diff
@@ -61,7 +63,7 @@
     private DocumentTypeConfigurationRepository documentTypeConfigurationRepositoryMock;
 
     @Mock
-    private WebHookService webHookServiceMock;
+    private CSOORDSService CSOORDSServiceMock;
```

Comment:
camelCase

Reply:
Since they are acronyms I usually don't camel case since they all represent a word. CSO and ORDS. But I can see the point and updated the variable name.